citizen/citizen.c

```diff
@@ -338,8 +378,10 @@ int getCitizenAge(List *list, int citizenId) {
     int currentMonth = current_time->tm_mon + 1;
```

Comment:
please follow the naming conventions on the variables, use snake_case

Reply:
still not followed

Reply:
Done, thanks.